src/util/rpc.js

```diff
@@ -4,7 +4,7 @@ class RPC {
   static SESSION_ID_HEADER = 'X-Transmission-Session-Id';
 
   constructor(onConnect = () => {}, onDisconnect = () => {}) {
-    this._url = '/transmission/rpc';
+    this._url = `/${document.location.pathname.split('/').filter((e) => e !== '')[0]}/rpc`;
```

Comment:
Thanks for your commit. What about using `lodash`?

```js
_.first(_.filter(document.location.pathname.split('/')))
```

Reply:
ok sure! give-me a minute